compat/maven-compat/src/main/java/org/apache/maven/repository/legacy/DefaultUpdateCheckManager.java

```diff
@@ -303,7 +301,7 @@ private Properties read(File touchfile) {
                 Properties props = new Properties();
 
                 try (FileInputStream in = new FileInputStream(touchfile)) {
-                    try (FileLock lock = in.getChannel().lock(0, Long.MAX_VALUE, true)) {
```

Comment:
the block is super small its **defacto** unused or im not Vincent Potucek.

<img width="1137" alt="image" src="https://github.com/user-attachments/assets/130bce87-a928-4ede-bf4e-bef2e88a50c7" />

Reply:
its is an error indication, therefore should be named according to convention.